check_critical_css.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+folder="build/critical-css/"
+file_list=$(find "$folder" -name '*.css')
+
+if [ "$file_list" == "" ]; then
+    echo "No files were generated in $folder"
+    exit 2
+fi
+
+echo "Files generated:"
+echo "$file_list"
+
+echo "$file_list" | while read filename; do
```

Comment:
We're expecting a set list, so probably ok to hard-code it: https://github.com/elifesciences/journal/blob/7b88378c1173d18f87fd16a1fddb397ead72622d/gulpfile.js#L177-L186

If the list ever changes, this will need to be updated too (but CI would alert us).

Reply:
`ci` doesn't seem to run critical-css, probably because it would add 60 seconds to each build?
https://github.com/elifesciences/builder-private/blob/master/salt/top.sls#L136-L176

Reply:
the first failure would then happen in end2end

Reply:
but if we can output just the list with a gulp task, we can use it as an input

Reply:
> the first failure would then happen in end2end

Which is always a little annoying, but this area doesn't change _much_ (so living with it might be ok).

Reply:
probably easier to read it from a file